scripts/gen_jvt.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # Fluster - testing framework for decoders conformance
-# Copyright (C) 2020, Fluendo, S.A.
+# Copyright (C) 2024, Fluendo, S.A.
```

Comment:
Check this out https://stackoverflow.com/questions/9039698/which-years-to-write-into-copyright-header.
The most common way for the year in copyright is one of
- first year that this code was worked on, e.g. 2020
- first to last year interval, e,g. 2020-2024

the reason for this is that although the file is new, the code was inherited and modified based on a previous implementation

Reply:
Done